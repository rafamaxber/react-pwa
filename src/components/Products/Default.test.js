import React from 'react';
import renderer from 'react-test-renderer';
import Product from './Default';

describe('Product', () => {
  it('Should match with snapshot', () => {
    const component = renderer.create(
      <Product></Product>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});