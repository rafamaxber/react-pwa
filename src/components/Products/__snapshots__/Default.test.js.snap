// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Should match with snapshot 1`] = `
<div
  className="sc-bdVaJa emYuWr"
>
  <div
    className="sc-bwzfXH uTxCW"
  >
    <img
      className="sc-htpNat KtFsv"
      src={undefined}
    />
  </div>
  <div
    className="sc-ifAKCX cFlEyZ"
  />
</div>
`;
